ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -94,20 +101,20 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
       title = 'Add new Institution';
       this.setState({institutionMode: InstitutionMode.ADD});
     }
-
   }
 
   // Filter out empty line or empty email addresses like <email1>,,<email2>
   // Confirm each email is a valid email using validate.js
   validateEmailAddresses() {
     const invalidEmailAddress = [];
-    const {emailAddresses} = this.state.institution;
+    const emailAddresses = getRegisteredTierEmailAddresses(this.state.institution);
     const updatedEmailAddress = emailAddresses.filter(
       emailAddress => {
         return emailAddress !== '' || !!emailAddress;
       });
 
-    this.setState(fp.set(['institution', 'emailAddresses'], updatedEmailAddress));
+    this.setState(fp.set(['institution', 'tierEmailAddresses'],
```

Comment:
Add a TODO comment (here and also for domains) that this will need an update for Controlled Tier

Reply:
Added. Then it would be the TODO everywhere. That todo is not quite useful as we probably will replace this page with a new one in https://precisionmedicineinitiative.atlassian.net/browse/RW-6933